src/test/compile-fail/range_traits-1.rs

```diff
@@ -17,77 +17,70 @@ use std::ops::*;
 struct AllTheRanges {
     a: Range<usize>,
     //~^ ERROR PartialOrd
```

Comment:
While you're at it, could you clean these up to use `//~|` instead of `//~^^^^^`?